### DISCUSSION TITLE
LDAP debug flag does not show debug infos

### DISCUSSION THREAD

**Discussion by ankyfive** [requarks/wiki#3577] [Error / Bug Report]
**Describe the bug**
I would like to use the debug flag for LDAP. I activated it it shows nothing in the console

**To Reproduce**
1. Go to developer tools
2. Click on ldap flag
3. Restart wiki.js
4. Login with AD and it does not work
5. See NO debug info just login password invalid

**Expected behavior**
Debug logs for LDAP

**Screenshots**

**Host Info (please complete the following information):**
 - OS: [Ubuntu 9.6]
 - Wiki.js version: [2.4.107]
 - Database engine: [postgres 9.7]

**Additional context**

**Comment by coxmic** [comment]
I also have this problem, LDAP debug flag does not work. 
OS: CentOS 7
DB: Postgres 11
App: Wiki.js 2.5.170

SQL Query Logging works.

**Comment by ankyfive** [comment]
I have been able to setup LDAP using ldapsearch binary
